### PR TITLE
AAdd script file and link it to html

### PR DESCRIPTION
**Pull request by christophschmitz**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+alert("Hello World");
```
Comment (LyriCaGitHubber) [inline index.js]:
```suggestion
let alertMessage = "Hello World";
alert(alertMessage);
```

**Review by LyriCaGitHubber — COMMENTED** [review]
siehe comment ;-)

File: index.html [hunk 2]
```diff
@@ -103,5 +103,6 @@ <h1>Tasks</h1>
         </svg>
       </button>
     </main>
+    <script src="index.js" defer="true"></script>
```
Comment (yoocly) [inline index.html]: No value required (following MDN)

```suggestion
    <script src="index.js" defer></script>
```

Comment (christophschmitz) [reply index.html]: Thank you!

**Pull request merged**